pub/htsget-openapi.yaml

```diff
@@ -0,0 +1,286 @@
+openapi: 3.0.0
+servers:
+  - description: htsget genomics api
+    url: https://virtserver.swaggerhub.com/brainkod/htsget/1.1.1
+info:
+  description: "This data retrieval API bridges from existing genomics bulk data transfers to a client/server model"
+  version: "1.1.1"
+  title: htsget
+  contact:
+    name: GA4GH
+    email: security-notification@ga4gh.org
+  license:
+    name: GPLv3
```

Comment:
[“The license information for the exposed API.”](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.0.md#info-object) Probably best to omit `license` for the time being.

Reply:
Yea, various parties may be allergic to GPL in particular -- omitting it is probably easier than selecting an alternative? :wink: